overtime_analyzer.py

```
@@ -0,0 +1,941 @@
+            # 시간 형식 검사 함수 (HH:mm 형식 고정)
+            def is_valid_time_format(time_str):
+                if not isinstance(time_str, str):
+                    return False
+                try:
+                    # HH:mm 형식 검사
+                    parts = time_str.strip().split(":")
+                    if len(parts) != 2:
+                        return False
+                    hour, minute = int(parts[0]), int(parts[1])
+                    return 0 <= hour < 24 and 0 <= minute < 60
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)